mainsite/catalog/templates/catalog/update_actual_content.html

```diff
@@ -0,0 +1,262 @@
+                  {% if field.errors %}
+                    <script type="text/javascript">
+                      document.getElementById("{{field.id_for_label}}").classList.add('is-invalid');
+                    </script>
+                  {% else %}
+                    <script type="text/javascript">
+                      document.getElementById("{{field.id_for_label}}").classList.add('is-valid');
+                    </script>
+                  {% endif %}
+              {% endfor %}
+            {% endif %}
+         </div>
+     </div>
+   </div>
+
+<!-- Used to prevent multiple submits -->
+<script type="text/javascript">
+  function ItemInputSubmit() {
+    document.getElementById("btnRealButton").disabled = true;
+    return true;
+  }
+</script>
+
+
+
+
+
+
+<template id="pic_upload_template">
+    <div class="row">
+        <div class="col-md-2">
+            <div style="height: 100%;  display: flex; flex-direction: column; flex-flow: column noWrap; justify-content: space-between; align-items: stretch;">
+                <button type="button" class="up_button" style="display: block;">↑</button>
+                <button type="button" class="delete_button">X</button>
+                <button type="button" class="down_button" style="display: block;">↓</button>
+            </div>
+        </div>
+
+        <div class="col-md-5">
+            <img height="200" width="200" src= {{ default_image_target }} >
+        </div>
+
+        <div class="col-md-5">
+            <label for="id_curr_picture">Picture:
+                <input type="file" name="curr_picture" accept="image/*" required="" class="form-control">
+            </label>
+
+        </div>
+    </div>
+</template>
+
+<template id="pic_upload_intermediate_template">
+    <div class="row">
+        <br> {% comment %} Yeah this should be done w/ CSS. #TODO {% endcomment %}
+        <div class="col-md-4"></div>
+        <div class="col-md-4">
+            <div style="width: 100%; padding-top: 20px; padding-bottom: 20px;"><button class="add_button" style="display: block; width: 100%; ">+</button></div>
+        </div>
+        <div class="col-md-4"></div>
+        <br>
+    </div>
+</template>
+
+<template id="super_secret_currently_uploaded_indicator">
+    <input type="hidden" name="currUpl" value="">
+</template>
+
+<template id="currently_uploaded_position_tracker_hack">
+    <input type="hidden" name="uplPos" value="">
+</template>
+
+<script type="text/javascript">
+
+    var table = null; //document.getElementById("blahblah");
+    var templ = null; //document.getElementById("pic_upload_template");
+    var inter = null; //document.getElementById("pic_upload_intermediate_template");
+    var sscui = null;
+    var cupth = null;
+
+
+    function getRowFromButton_3(el) {
+        return el.parentNode.parentNode.parentNode;
+    };
+    function getRowFromButton_I(el) {
+        return el.parentNode.parentNode.parentNode;
+    };
+
+    /*function resetEnablers() {
```

Comment:
Probably should remove commented out code.

Reply:
✔️